### PR TITLE
Removed the KnexListAdapter.createAdjacencyTable method.

### PR DESCRIPTION
**Pull request by timleslie**
This is part of a series of refactorings to move towards a more consistent relationship model.

This PR doesn't make any functional changes.

### CODE REVIEW

File: packages/adapter-knex/lib/adapter-knex.js [hunk 1]
```diff
@@ -102,6 +102,57 @@ class KnexAdapter extends BaseKeystoneAdapter {
     return fkResult;
   }
 
+  async _createAdjacencyTable({ tableName, relationshipFa, leftListAdapter }) {
+    // Create an adjacency table for the (many to many) relationship field adapter provided
+    const dbAdapter = this;
+    try {
+      console.log(`Dropping table ${tableName}`);
+      await dbAdapter.schema().dropTableIfExists(tableName);
```
Comment (MadeByMike) [inline packages/adapter-knex/lib/adapter-knex.js]: Do you don't want to drop the table every time? Won't this loose some relationships?

File: packages/adapter-knex/lib/adapter-knex.js [hunk 2]
```diff
@@ -102,6 +102,57 @@ class KnexAdapter extends BaseKeystoneAdapter {
...
+    } catch (err) {
+      console.log('Failed to drop');
+      console.log(err);
+      throw err;
+    }
+
+    // To be clear..
+    const leftPkFa = leftListAdapter.getPrimaryKeyAdapter();
```
Comment (MadeByMike) [inline packages/adapter-knex/lib/adapter-knex.js]: If we're going to be clear can we be clearer? `leftPrimaryKey(what is 'Fa')`

File: packages/adapter-knex/lib/adapter-knex.js (hunk 1 above)
Comment (timleslie) [reply packages/adapter-knex/lib/adapter-knex.js]: If we're at this point of the code it's assumed that we're recreating the database, so we cleanup the table we're about to create just to be 100% certain we're starting from a clean slate.

File: packages/adapter-knex/lib/adapter-knex.js (hunk 2 above)
Comment (timleslie) [reply packages/adapter-knex/lib/adapter-knex.js]: The cleanliness of this code is out of scope for this PR. It's all just moving around here, and it's almost definitely going to change in future PRs.

**Pull request merged**